fulltime-developer.md

```diff
@@ -1,39 +1,37 @@
 # Mr. Henry is looking for a fulltime developer
 
-*At Mr. Henry, we sail against the current of the wild web development seas.*
+We are looking for a fulltime developer to join our team.
 
-Let aside the bad metaphors, we're working on our own way to build for the web. Instead of hopping on every new framework that pops up, we choose to take a step back and work on something stubborn & better.
-
-Based on new, modern, and future-proof web standards, we're trying to build websites that work, everywhere, for everyone. For every *Shiny New Thing™*, we want to raise a thing that *Just Works™*.
+## Who we are
 
-If this tickles your curiosity, we'd like to talk to you.
-If you think we're 100% in the wrong, we'd still like to talk to you.
+Mr. Henry is an agency with a small team of passionate experts.  
+We're a mix of developers and designers working on online and offline projects.  
 
-## Who we are
+We work on branding, design and web development.  
```

Comment:
```suggestion
We work on design, strategy and technology.  
```

Terms taken from https://www.mrhenry.be/work/

Reply:
I strongly prefer `web development` over `technology`.
Given that we do only web development and nothing beyond that :D